packages/api-server/src/app.ts

```diff
@@ -51,6 +51,12 @@ const app: FastifyPluginAsync<AppOptions, RawServerDefault, JsonSchemaToTsProvid
     opts
 ): Promise<void> => {
 
+  // Load Auth0
+  fastify.register(require("fastify-auth0-verify"), {
+    domain: process.env.AUTH0_DOMAIN,
```

Comment:
Please add the environment to env.ts and https://github.com/pingcap/ossinsight/blob/1557f0e1a122e1684317bbdbb3c28e7505a3244f/packages/api-server/src/app.ts#L21

And register the plugin in the `plugins` folder.